packages/contracts/contracts/FeeManager.sol

```diff
@@ -0,0 +1,57 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.3;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+contract FeeManager is Ownable {
+    uint256 public version = 1;
+
+    enum ExemptionType {
+        None,
+        Lending,
+        Subscription
+    }
+
+    struct FeeExemption {
+        ExemptionType exemptionType;
+        uint256 endDate;
+    }
+
+    uint256 public feePercentage = 0;
```

Comment:
I think the fee should be in basis points. This wouldn't allow for something like 0.75%